resources/alerts.yaml

```diff
@@ -5,29 +5,15 @@ metadata:
   name: cassandra-load
 spec:
   formula: |
-    var period = 30m
-    var every = 5m
-    var data_cassandra_load = stream
-        |from()
-            .measurement('cassandra_load')
-        |where(lambda: "max" > 1073741824)
-        |window()
-            .period(period)
-            .every(every)
-
-    var trigger_load_difference = data_cassandra_load
-        |eval(lambda: 100.0 - float("min")*100.0/float("max"))
-            .as('percent_difference')
-        |alert()
-            .message('{{ .Level }} / cassandra: storage load is different between nodes')
-            .crit(lambda: "percent_difference" > 20.0)
-            .critReset(lambda: "percent_difference" < 10.0)
-            .stateChangesOnly(1h)
-            .details('''
-    <b>{{ .Message }}</b>
-    <p>Level: {{ .Level }}</p>
-    <p>Storage load difference(%): {{ index .Fields "percent_difference" | printf "%0.2f" }}</p>
-    ''')
-            .email()
-            .log('/var/lib/kapacitor/logs/cassandra_load.log')
-            .mode(0644)
+    groups:
+    - name: cassandra-load-alert
+      rules:
+      - record: cassandra_load:percent_diff
+        expr: 100.0 - ((cassandra_load_min * 100.0) / cassandra_load_max > 1073741824)
```

Comment:
I don't the expression is correct here. Can you elaborate on it?
The rule should be `compare the difference between the minimum and maximum but only if the maximum value is higher than 1GB(to avoid flaky results for an empty database)`.

Reply:
In your record, you are comparing result of `(cassandra_load_min * 100.0) / cassandra_load_max` to 1073741824
https://prometheus.io/docs/prometheus/latest/querying/operators/#binary-operator-precedence